### PR TITLE
[OptionsResolver] removed deprecated functionality

### PR DESCRIPTION
**Pull request by Tobion**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | yes |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/OptionsResolver/OptionsResolver.php [hunk 1]
```diff
@@ -424,18 +424,6 @@ public function setNormalizer($option, \Closure $normalizer)
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function setNormalizers(array $normalizers)
```
Comment (Tobion) [inline src/Symfony/Component/OptionsResolver/OptionsResolver.php]: we need to add trigger errors for that in 2.7 which has been forgotten

File: src/Symfony/Component/OptionsResolver/OptionsResolver.php [hunk 2]
```diff
@@ -456,21 +444,12 @@ public function setNormalizers(array $normalizers)
...
      */
-    public function setAllowedValues($option, $allowedValues = null)
+    public function setAllowedValues($option, $allowedValues)
     {
         if ($this->locked) {
             throw new AccessException('Allowed values cannot be set from a lazy option or normalizer.');
         }
 
-        // BC
-        if (is_array($option) && null === $allowedValues) {
```
Comment (Tobion) [inline src/Symfony/Component/OptionsResolver/OptionsResolver.php]: needs trigger error as well in 2.7

File: src/Symfony/Component/OptionsResolver/OptionsResolver.php (hunk 1 above)
Comment (Tobion) [inline src/Symfony/Component/OptionsResolver/OptionsResolver.php]: done #13809

**Comment by stof** [comment]
You forgot to remove the OptionsResolverInterface here

**Comment by Tobion** [comment]
@stof it has already been removed by another PR

**Pull request merged**